packages/uswds-elements/lib/_normalize.scss

```diff
@@ -349,3 +349,8 @@ template {
 [hidden] {
   display: none;
 }
+
+[hidden="until-found"] {
+  display: revert;
+  content-visibility: hidden;
```

Comment:
Curious - does this rule need to be explicitly defined? It looks like the browser implicitly applies it with `hidden="until-found"` and removing it doesn't seem to have an affect.  

> hidden=until-found applies the content-visibility:hidden CSS property instead of the display:none property that the regular hidden attribute applies. [^1]

[^1]: [developer.chrome.com](https://developer.chrome.com/docs/css-ui/hidden-until-found#caveats)

```suggestion
```

Reply:
@amyleadem this was added explicitly for clarity. Especially as this is a new feature. Please let me know if you'd like me to remove.